### ISSUE TITLE
Question: size of logits

### ISSUE THREAD

**Issue by HWZX** [kyzhouhzau/BERT-NER#4]
hi kyzhouhzau~

thank you for this project
I have a question about the size of num_labels in BERT_NER.py line470.
https://github.com/kyzhouhzau/BERT-NER/blob/2a9fcb8f857e9ac0bbd166437a4c9fe7bbc7d0c3/BERT_NER.py#L470

num_labels=len(label_list)+1,
why you let the size of logits one bigger than the size of label_list?
thank you so much!

**Comment by kyzhouhzau** [comment]
@HWZX 
Because I encode the label beginning from one, but I used 0 to padding the length. Here I did not use the masking method to calculate the loss, so zero is another label. But in the final evaluation of the result macro-f did not count the padding part.
